pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -505,6 +524,44 @@ func (r *ReconcileIBMBlockCSI) reconcileServiceAccount(instance *ibmblockcsi.IBM
 	return nil
 }
 
+func (r *ReconcileIBMBlockCSI) getRestartedAtAnnotation(Annotations map[string]string) (string, string){
+	restartedAt := fmt.Sprintf("%s/restartedAt", oconfig.APIGroup)
+	for key, element := range Annotations {
+		if key == restartedAt {
+			return key, element
+		}
+	}
+	return "", ""
+}
+
+func (r *ReconcileIBMBlockCSI) getControllerK8sObject(instance *ibmblockcsi.IBMBlockCSI) (*appsv1.StatefulSet, error) {
+			controllerStatefulset := &appsv1.StatefulSet{}
+			err := r.client.Get(context.TODO(), types.NamespacedName{
+				Name:      oconfig.GetNameForResource(oconfig.CSIController, instance.Name),
+				Namespace: instance.Namespace,
+			}, controllerStatefulset)
+
+			return controllerStatefulset, err
+}
+
+func (r *ReconcileIBMBlockCSI) getNodeK8sObject(instance *ibmblockcsi.IBMBlockCSI) (*appsv1.DaemonSet, error) {
+	node := &appsv1.DaemonSet{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{
+		Name:      oconfig.GetNameForResource(oconfig.CSINode, instance.Name),
+		Namespace: instance.Namespace,
+	}, node)
+
+	return node, err
+}
+
+func (r *ReconcileIBMBlockCSI) doesControllerPodsRunning(instance *ibmblockcsi.IBMBlockCSI, controller *appsv1.StatefulSet) bool {
+	return controller.Status.ReadyReplicas == controller.Status.Replicas
+}
+
+func (r *ReconcileIBMBlockCSI) doesNodePodsRunning(instance *ibmblockcsi.IBMBlockCSI, node *appsv1.DaemonSet) bool {
+	return node.Status.DesiredNumberScheduled == node.Status.NumberAvailable
+}
```

Comment:
why do they need the `instance`?
boolean functions usually start with `is`, and in this case the high-level question is `is<Component>Ready`

Reply:
done

Reply:
could you please renaming the functions to `isControllerRead` and `isNodeReady` respectively?

Reply:
done